src/libraries/Common/tests/StreamConformanceTests/System/IO/StreamConformanceTests.cs

```diff
@@ -1635,8 +1632,8 @@ protected IEnumerable<(Stream writeable, Stream readable)> GetReadWritePairs(Str
             streams.Stream2.CanRead && streams.Stream2.CanWrite;
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/67853", TestPlatforms.tvOS)]
         [SkipOnPlatform(TestPlatforms.LinuxBionic, "SElinux blocks UNIX sockets")]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS, "iOS/tvOS blocks binding to UNIX sockets")]
```

Comment:
Should we make this a `ConditionalFact(typeof(Socket), nameof(Socket.OSSupportsUnixDomainSockets))` instead?

We probably should also make Linux Bionic return false for that property.

Reply:
Bionic _does_ support UNIX sockets, but it explicitly doesn't permit them in the `/data/local/tmp` directory we use for our test suite (the only folder, more or less, one can use when just using `adb shell` without the apk infrastructure around it)